case-lib/lib.sh

```diff
@@ -1187,3 +1187,39 @@ perf_analyze()
     fi
 }
 
+# test-mic_privacy.sh needs to control mic privacy settings (on/off)
+# needs usbrelay package: https://github.com/darrylb123/usbrelay
+# param1: switch name
+# param2: switch state
+usbrelay_switch()
+{
+    local switch_name=$1
+    local state=$2
+    dlogi "Setting usbrelay switch $switch_name to $state."
+    usbrelay "$switch_name=$state" >/dev/null 2>&1 || {
```

Comment:
do you think `usbrelay`-s output with possible errors is not useful for diagnostics and troubleshooting ?
```suggestion
    usbrelay "$switch_name=$state" 2>&1 || {
```

Reply:
Discarding stderr is wrong 99.999% of the time. The remaining 0.001% requires a comment with a very strong rationale.

That's because 99.999% of the time, developers use stderr because they think the error is so bad that the message should never be discarded.

Discarding stderr even more wrong in TEST code! It's a really bad code smell.